tracer/test/Datadog.Trace.Tests/ServerlessTests.cs

```diff
@@ -0,0 +1,75 @@
+// <copyright file="ServerlessTests.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.IO;
+
+using Datadog.Trace.ClrProfiler.ServerlessInstrumentation;
+using FluentAssertions;
+using Xunit;
+
+namespace Datadog.Trace.Tests
+{
+    public class ServerlessTests
+    {
+        [Fact]
+        public void IsRunningInLambdaFalseNoFileAndNoEnvironmentVariable()
+        {
+            Environment.SetEnvironmentVariable("AWS_LAMBDA_FUNCTION_NAME", string.Empty);
```

Comment:
You will likely want to capture the value _before_ you set it, run the test, then set it back afterwards. Otherwise, you may end up with flaky tests down the line!